src/download-cli.ts

```diff
@@ -1,6 +1,7 @@
 import { addPath, debug } from '@actions/core';
 import { exec } from '@actions/exec';
 import { find, downloadTool, cacheDir, cacheFile, findAllVersions } from '@actions/tool-cache';
+// eslint-disable-next-line @typescript-eslint/no-require-imports
 import fs = require('fs');
```

Comment:
So I tried fixing this by replacing it with:
```ts
import fs from 'fs';
// also tried this
import fs from 'node:fs';
```
But these both resulted in [this error](https://github.com/fossas/fossa-action/actions/runs/14265605972/job/39986615205#step:6:7). Just gonna disable the lint here and move on.